github.go

```diff
@@ -126,7 +127,7 @@ func (m *GithubClient) ListOpenPullRequests() ([]*PullRequest, error) {
 		"prFirst":         githubv4.Int(100),
 		"prStates":        []githubv4.PullRequestState{githubv4.PullRequestStateOpen},
 		"prCursor":        (*githubv4.String)(nil),
-		"commitsLast":     githubv4.Int(1),
+		"commitsLast":     githubv4.Int(100),
```

Comment:
This is intended to be just the very last commit (i.e. 1 and not 100) since the function is just used for `check`.

Reply:
Yes of course. Sorry was confused here because of getting 100 changed files in one go. Will fix. Thanks for catching that.